### PR TITLE
gluon-status-page: add gateway_nexthop to status page statistics

### PR DESCRIPTION
**Pull request by lemoer**

### CODE REVIEW

**Comment by lemoer** [comment]
Screenshot from 2019-11-24 01-31-54

**Comment by mweinelt** [comment]
Maybe just

```
Gateway  $MAC
         via $NEXTHOP
```

File: package/gluon-status-page/javascript/status-page.js [hunk 1]
```diff
@@ -112,6 +112,17 @@
 		'bytes': function(bytes) {
 			return prettyBytes(bytes);
 		},
+		'neighbour': function(addr) {
+			for (var i in interfaces) {
+				var iface = interfaces[i];
+				var neigh = iface.get_neigh(addr);
```
Comment (mweinelt) [inline package/gluon-status-page/javascript/status-page.js]: Maybe improvide readability like this, you're not doing any error handling at this point anyway.
```
			for (var interface in interfaces) {
				var neigh = interfaces[interface].get_neigh(addr);
```

Which leads me to the question: How does it look when the node is offline?

**Comment by lemoer** [comment]
Screenshot from 2019-11-24 02-21-07

**Comment by mweinelt** [comment]
I accidentally pushed these commits to master and reverted them shortly after. GitHub won't let me reopen this pull requests,.

**Comment by rotanid** [comment]
@mweinelt what stands against merging this?

**Comment by mweinelt** [comment]
It didn't work when I tested it locally.

**Comment by lemoer** [comment]
@mweinelt Can you please post a screenshot and/or explain, what is broken?

**Comment by mweinelt** [comment]
I linked you the nodes status-page recently. If you don't have that anymore I need to rebuild and redeploy.

**Comment by mweinelt** [comment]
http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/cgi-bin/status

```
TypeError: "u[n] is not a function"
    <anonymous> http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
    <anonymous> http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
    onmessage http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
status-page.js:1:1776
    <anonymous> http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
    <anonymous> http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
    onmessage http://[2001:67c:2ed8:1001:e695:6eff:fe41:f7a0]/static/status-page.js:1
```

**Pull request merged**